client/src/app/site/motions/services/amendment-list-pdf.service.ts

```diff
@@ -108,7 +109,7 @@ export class AmendmentListPdfService {
 
         const table: object = {
             table: {
-                widths: ['auto', 'auto', 'auto', '*', 'auto'],
+                widths: ['auto', '*', '*', 'auto', '*'],
```

Comment:
I will change it back to `['auto', '*', '*', 'auto', '*']` which looks better for long lists.